app/com/arpnetworking/rollups/RollupGenerator.java

```diff
@@ -281,22 +322,36 @@ private LastDataPointMessage buildLastDataPointResponse(
         if (failure != null) {
             builder.setFailure(failure);
         } else {
```

Comment:
I changed the following section to use `streams/map/flatMap` because the chain of `get()` calls to nullable return values gave me anxiety